server/routes/bookedtoday/summaryWithRecordController.test.ts

```diff
@@ -62,66 +62,27 @@ describe('GET /prisoner/:id/summary-with-record', () => {
   })
 
   describe('prisoner image', () => {
-    it('should render prisoner image when Prison Number provided', () => {
+    it('should render prisoner image with single potential match Prison Number', () => {
       return request(app)
         .get('/prisoners/1111-1111-1111-1111/summary-with-record')
         .expect(200)
         .expect('Content-Type', 'text/html; charset=utf-8')
         .expect(res => {
           const $ = cheerio.load(res.text)
-          expect($('[data-qa=prisoner-image]').attr('src')).toEqual('/prisoners/A1234AB/image')
-        })
-    })
-
-    it('should render placeholder image when no Prison Number provided', () => {
-      expectedArrivalsService.getArrival.mockResolvedValue(withMatchType(createArrival({ prisonNumber: null })))
-
-      return request(app)
-        .get('/prisoners/1111-1111-1111-1111/summary-with-record')
-        .expect(200)
-        .expect('Content-Type', 'text/html; charset=utf-8')
-        .expect(res => {
-          const $ = cheerio.load(res.text)
-          expect($('[data-qa=prisoner-image]').attr('src')).toEqual('/assets/images/placeholder-image.png')
+          expect($('[data-qa=prisoner-image]').attr('src')).toEqual('/prisoners/A5534HA/image')
         })
     })
   })
 
   describe('caption', () => {
-    it('should render both PNC Number and Prison Number when both are given', () => {
-      return request(app)
-        .get('/prisoners/1111-1111-1111-1111/summary-with-record')
-        .expect(200)
-        .expect('Content-Type', 'text/html; charset=utf-8')
-        .expect(res => {
-          const $ = cheerio.load(res.text)
-          expect($('span.govuk-caption-l').text().trim()).toStrictEqual('Prison number: A1234AB | PNC: 01/98644M')
-        })
-    })
-
-    it('should render only Prison Number when only Prison Number is given', () => {
-      expectedArrivalsService.getArrival.mockResolvedValue(withMatchType(createArrival({ pncNumber: null })))
-
-      return request(app)
-        .get('/prisoners/1111-1111-1111-1111/summary-with-record')
-        .expect(200)
-        .expect('Content-Type', 'text/html; charset=utf-8')
-        .expect(res => {
-          const $ = cheerio.load(res.text)
-          expect($('span.govuk-caption-l').text().trim()).toStrictEqual('Prison number: A1234AB')
-        })
-    })
-
-    it('should render only PNC Number when only PNC Number is given', () => {
-      expectedArrivalsService.getArrival.mockResolvedValue(withMatchType(createArrival({ prisonNumber: null })))
-
+    it('should render both PNC Number and Prison Number from single potential match', () => {
       return request(app)
```

Comment:
removed test as will always display both prison number and PNC number for single match